src/app/components/demo-components.service.ts

```diff
@@ -454,10 +454,18 @@ export class SkyDemoComponentsService {
         name: 'Text expand',
         icon: 'text-height',
         // tslint:disable-next-line
-        summary: `The text expand component truncates long text and provides a link to expand the text.`,
+        summary: `The text expand component truncates long blocks of text with an ellipsis and a link to expand the full text.`,
         url: '/components/text-expand',
         getCodeFiles: () => this.getDemoFiles('SkyTextExpandDemoComponent')
       },
+      {
+        name: 'Text expand repeater',
+        icon: 'list',
+        // tslint:disable-next-line
+        summary: `The text expand repeater component truncates a list of items and initially displays a limited number of items.`,
+        url: '/components/text-expand-repeater',
+        getCodeFiles: () => this.getDemoFiles('SkyTextExpandDemoComponent')
```

Comment:
This will need updated once the new demo is released.

Reply:
Yep. I just hadn't updated it yet so that I could serve locally. I'll go ahead and make the change and then test it after we release the demo and import it.